app/src/main/java/de/danoeh/antennapod/fragment/ExternalPlayerFragment.java

```diff
@@ -200,7 +200,6 @@ private void updateUi(Playable media) {
                 .into(imgvCover);
 
         if (controller != null && controller.isPlayingVideoLocally()) {
-            butPlay.setVisibility(View.GONE);
```

Comment:
i have removed this line  because whenever a video is played in AntennaPod this line was making play button invisible